### PR TITLE
fix: Statistics selection

### PR DESCRIPTION
**Pull request by FedeMadoery**
# Description

### CODE REVIEW

File: src/hooks/nextjs/useStatisticsVisibility.tsx [hunk 1]
```diff
@@ -22,20 +22,23 @@ export default function useStatisticVisibility(
 
   useEffect(() => {
     if (stack.length === 0) {
+      console.log('useEffect')
```
Comment (ramabit) [inline src/hooks/nextjs/useStatisticsVisibility.tsx:25]: please remove logs

File: src/hooks/nextjs/useStatisticsVisibility.tsx [hunk 2]
```diff
@@ -22,20 +22,23 @@ export default function useStatisticVisibility(
...
       // Fulfill helper stack to keep just 2 items select
       Object.keys(statisticVisibility).map((keyName) => {
         if (statisticVisibility[keyName]) setStack((prevState) => [keyName, ...prevState])
       })
     }
   }, [stack.length, statisticVisibility])
 
+  console.log('stack', stack)
```
Comment (ramabit) [inline src/hooks/nextjs/useStatisticsVisibility.tsx:33]: please remove logs

**Review by ramabit — CHANGES_REQUESTED** [review]

File: src/hooks/nextjs/useStatisticsVisibility.tsx (hunk 2 above)
Comment (FedeMadoery) [reply src/hooks/nextjs/useStatisticsVisibility.tsx:33]: Yeah, but i was waiting the confirmation about it is working for you guys too hehehe

Comment (ramabit) [reply src/hooks/nextjs/useStatisticsVisibility.tsx]: from I what I tested, it worked good for me

**Review by ramabit — APPROVED** [review]

**Pull request merged**